cuda_python/docs/build_all_docs.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -ex
+
+# build cuda-python docs
+rm -rf build
+./build_docs.sh
+
+# build cuda-bindings docs
+CUDA_BINDINGS_PATH=build/html/cuda-bindings
+mkdir -p $CUDA_BINDINGS_PATH
+pushd .
+cd ../../cuda_bindings/docs
+rm -rf build
+./build_docs.sh
+cp -r build/html/* "$(dirs +1)"/$CUDA_BINDINGS_PATH
```

Comment:
>cp: target '~/tip/public/leofang/cuda-python/cuda_python/docs/build/html/cuda-bindings' is not a directory

Tilde expansion doesn't work for me. Can you replace it with `$(dirs -l +1)`?

Reply:
Good catch, fixed in commit fa8d482. I was developing on a shared machine where I could not put large files under `~/`, thus missing this.